Update README.md (#104)
dinngo/protocolink-contract: README.md

## Overview

- Protocolink is a router system which consolidates protocol interactions within a secure Router/Agent architecture in a single transaction.

> - Protocolink is proficient in processing ERC20, ERC721, ERC1155 and lending positions.

Reviewer: ```suggestion
- Protocolink is proficient in processing ERC-20, ERC-721, ERC-1155 and lending positions.
```
To be consistent with the below `ERC-20 tokens are transferred...`

> - Protocolink is protocol-agnostic. All protocol-related code (eg., the [Protocolink Logic](https://github.com/dinngo/protocolink-logics)) is defined outside of the contracts. Protocolink also offers an [API](https://app.gitbook.com/o/-MJ0tv29EsBtmEHq0z5Q/s/HuiitAfgh7HXRqRn5di2/integrate-api/overview) and a [SDK](https://app.gitbook.com/o/-MJ0tv29EsBtmEHq0z5Q/s/HuiitAfgh7HXRqRn5di2/integrate-js-sdk/overview) for developers to create transactions.

Reviewer: ```suggestion
- Protocolink is protocol-agnostic. All protocol-related code is defined in the [protocolink-logics](https://github.com/dinngo/protocolink-logics) repository separate from the contracts. Protocolink also offers an [API](https://docs.protocolink.com/integrate-api/overview) and a [SDK](https://docs.protocolink.com/integrate-js-sdk/overview) for developers to create transactions.
```
